### PR TITLE
Skill integration test implementation

### PR DESCRIPTION
**Pull request by Evgeniy-Sinicin**

### CODE REVIEW

File: InternshipBE/DAL/Repositories/EFUnitOfWork.cs [hunk 1]
```diff
@@ -30,6 +30,8 @@ public class EFUnitOfWork : IUnitOfWork
         private IValidator<InterviewInvite> _interviewInviteValidator;
         private IValidator<Team> _teamValidator;
 
+        public ApplicationDbContext Context => _db;
```
Comment (Evgeniy-Sinicin) [inline InternshipBE/DAL/Repositories/EFUnitOfWork.cs:33]: I need have access to db to execute BeginTransaction() method 🙄

File: InternshipBE/Tests/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using DAL.Repositories;
+using DAL.Validators;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Tests
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
```
Comment (Evgeniy-Sinicin) [inline InternshipBE/Tests/Startup.cs:16]: Configured services for XUnit.DependencyInjection nuget packege

**Review by Xodoc — APPROVED** [review]

File: InternshipBE/Tests/IntegrationTests/Services/SkillServiceTest.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            Assert.Equal(expectedSkillDtos, actualSkillDtos, new SkillDTOEqualityComparer());
+
+            await transaction.RollbackAsync();
+        }
+
+        [Fact]
+        public async void GetSkillsByStackTypesAsync_WhenStacksAreBackEndAndFrontEnd_GettingBackEndAndFrontEndSkills()
+        {
+            var stacks = new StackType[] { StackType.FrontEnd, StackType.BackEnd };
+            var expectedSkillDtos = _skillDtos.Where(x => stacks.Contains(Enum.Parse<StackType>(x.StackType))).ToList();
```
Comment (Xodoc) [inline InternshipBE/Tests/IntegrationTests/Services/SkillServiceTest.cs:59]: DTO! not Dto)

Comment (Evgeniy-Sinicin) [reply InternshipBE/Tests/IntegrationTests/Services/SkillServiceTest.cs:59]: image

Comment (Xodoc) [reply InternshipBE/Tests/IntegrationTests/Services/SkillServiceTest.cs:59]: ok, what do you say now?))) This is your code))
image

Comment (Evgeniy-Sinicin) [reply InternshipBE/Tests/IntegrationTests/Services/SkillServiceTest.cs:59]: ahaha, chill - chill 🍦
we learned about the code style after badly named dtos 😂

**Review by romanstat — APPROVED** [review]

**Review by vldmrklnk — APPROVED** [review]

**Review by ihhyy — APPROVED** [review]

**Pull request merged**